### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by Dillowplays** [UseInterstellar/Interstellar#517] [General]
Can someone send me the link the a site, if you have the nasa one link. That would be great! @IncognitoTGT @Bigfoot9999

**Comment by MadjikDotPng** [comment]
Discussions are not a place to ask for links. You can join our [discord](https://discord.gg/Interstellar) to obtain links or visit our [main demo website](https://gointerstellar.app/)!

**Comment by SchoolTARMaster** [comment]
I still have a link:
<a href="https://scienceresult.vercel.app">Here</a>.
